ansible_runner/utils/__init__.py

```diff
@@ -464,3 +465,17 @@ def get_executable_path(name):
     if exec_path is None:
         raise ConfigurationError(f"{name} command not found")
     return exec_path
+
+
+# signal handler for default cancel_callback
+class SignalHandler:
```

Comment:
This basically reimplements `threading.Event`. That class cloud be used instead.

Reply:
✅